scripts/fuzz/generate_decoding_fuzz_corpus.py

```
@@ -214,6 +221,12 @@ def main() -> int:
         "chain": phase12,
     }
     write_json(phase30_path, phase30_input)
+
+    if not PHASE29_CURATED_CORPUS.is_file():
+        raise SystemExit(f"missing checked-in Phase 29 corpus seed: {PHASE29_CURATED_CORPUS}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)